core-lib/Kernel.ns

```diff
@@ -410,8 +413,18 @@ class Kernel vmMirror: vmMirror = Object <: Value (
       ^ self substringFrom: argument to: argument
     )
 
+    public do: block = (
```

Comment:
Why do we need #do: on strings?
Also, could you please add a test for any addition to Kernel.ns?